code/workspaces/infrastructure-api/src/controllers/volumeController.spec.js

```diff
@@ -89,6 +89,13 @@ describe('Volume Controller', () => {
       await executeCreateValidator(requestBody);
       expectValidationError('volumeSize', 'Volume Size must be an integer between 5 and 200');
     });
+
+    it('should validate the type is between 1 and 2', async () => {
```

Comment:
I don't understand why this checking the type be between two numeric values, should this be checking they are of the new string values for the type?

Reply:
Thanks, corrected.
(That test description comes from an earlier commit where 1=glusterfs,2=nfs, before I decided to change the Mongo storage to save as string enums).